src/app/config.json

```diff
@@ -1,30 +1,66 @@
 {
     "dev": {
-        "isTesting": false,
-        "masterEndpointConfig": {
+        "IS_TESTING": false,
+        "MASTER_ENDPOINT_CONFIG": {
```

Comment:
No. My personal opinion on this is all caps make things like constants. And your argument to introducins this is?